app.js

```diff
@@ -0,0 +1,16 @@
+const hamburger = document.querySelector('.hamburger');
+const menu = document.querySelector('.nav-menu');
+const body = document.getElementById('body');
+
+hamburger.addEventListener('click', () => {
+  hamburger.classList.toggle('active');
+  menu.classList.toggle('active');
+  body.classList.toggle('active');
+});
+
+document.querySelectorAll('.nav-links').forEach((i) => {
+  i.addEventListener('click', () => {
+    hamburger.classList.remove('active');
+    menu.classList.remove('active');
+  });
+});
```

Comment:
- [ ] Ouch! you seem to be missing one of the requirements that states: "When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed." So please make sure you implement that in your JS file here. ✔